public/static/docs/command-reference/fetch.md

```diff
@@ -185,6 +185,13 @@ $ tree .dvc
 ├── ...
 ```
 
+> The cache file with `.dir` extension is a special text file that contains the
```

Comment:
probably not the purpose of fetch to explain exact cache structure, I keep it simpler

Reply:
You're right. I'm just going to move the "Note that the `.dvc/cache` directory was created and populated." note from the command sample to a regular paragraph and leave the link to /doc/user-guide/dvc-files-and-directories#structure-of-cache-directory for more info... ⏳

Reply:
Done as well. I also found this note in another few docs and removed/updated accordingly.

See 13e516ec

Reply:
so, my point is more about the `.dir` explanation. We don't need to have a note about it. It's fine to mention about cache structure.

Reply:
Yep, I got it. I did remove the notes about .dir cache file and just left a note/link to cache structure 🙂